easybuild/framework/easyblock.py

```diff
@@ -4422,6 +4426,70 @@ def print_dry_run_note(loc, silent=True):
     dry_run_msg(msg, silent=silent)
 
 
+def copy_build_dirs_logs_failed_installs(application_log, silent, app, easyconfig):
+    """
+    Copy build directories and log files for failed installation (if desired)
+    """
+    # there may be multiple log files, or the file name may be different due to zipping
+    logs = glob.glob(f"{application_log}*")
+
+    datestamp = time.strftime('%Y%m%d')
+    timestamp = time.strftime('%H%M%S')
+    salt = ''.join(random.choice(ascii_letters) for i in range(5))
+    unique_subdir = f'{datestamp}-{timestamp}-{salt}'
+
+    operation_args = []
+
+    logs_path = get_failed_installs_logs_path(easyconfig)
+    if logs_path:
```

Comment:
Just to be sure and clear:
```suggestion
    if logs and logs_path:
```